CONTRIBUTING.md

```diff
@@ -92,3 +92,17 @@ rsqZ
 =wHOK
 -----END PGP PUBLIC KEY BLOCK-----
 ```
+
+## Contributor License Agreement
```

Comment:
```suggestion
## Contributor Licence Agreement
```
Proper English ;)

Reply:
Nice try, but I trust wikipedia on that ;) 
https://en.wikipedia.org/wiki/Contributor_License_Agreement

Reply:
It's because Wikipedia is written in American...